fsw/mcp750-vxworks/src/cfe_psp_support.c

```diff
@@ -115,7 +115,7 @@ void CFE_PSP_Restart(uint32 reset_type)
 void CFE_PSP_Panic(int32 ErrorCode)
 {
     printf("CFE_PSP_Panic Called with error code = 0x%08X. Exiting.\n", (unsigned int)ErrorCode);
-    exit(-1); /* Need to improve this */
+    exit(EXIT_FAILURE); /* Need to improve this */
```

Comment:
Recommend removing comment and rebasing.  Then should be good to go.

NOTE: If anyone knows what improvement this was referring to I suggest writing an issue.  Linux aborts to get a core file, RTEMS just exits.

Reply:
Jake @skliper - Rebased to main and removed the comment.
Cheers